packages/airnode-utilities/src/logging.ts

```diff
@@ -0,0 +1,133 @@
+import { formatDateTimeMs } from './utils';
+import { LogLevel, LogOptions, PendingLog, LogMetadata, LogConfig } from './types';
+
+const logLevels: { readonly [key in LogLevel]: number } = {
+  DEBUG: 0,
+  INFO: 1,
+  WARN: 2,
+  ERROR: 3,
+};
+
+export const logger = {
+  buildBaseOptions: (config: LogConfig, meta: LogMetadata) => ({
+    format: config.nodeSettings.logFormat,
+    level: config.nodeSettings.logLevel,
+    meta,
+  }),
+  log: (message: string, options?: LogOptions) => {
```

Comment:
I wonder if we could allow creating a logger with options only once, so they don't have to be passed on every call - but that can be done as a separate issue.

Reply:
This would be wonderful.